net/lora-gateway-hal/Makefile

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (C) 2019 Xue Liu <liuxuenetmail@gmail>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lora-gateway-hal
+PKG_VERSION:=5.0.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/Lora-net/lora_gateway/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=1a0447d5e8183d08e6dce5f739f6872b9c57824b98f4078830d5ee21b15782c1
+PKG_MAINTAINER:=Xue Liu <liuxuenetmail@gmail.com>
+PKG_LICENSE_FILES:=LICENSE
+PKG_BUILD_DIR:=$(BUILD_DIR)/lora_gateway-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libloragw
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=Driver/HAL library for Semtech SX1301
+  URL:=http://www.semtech.com/wireless-rf/lora.html
+endef
+
+define Package/libloragw/description
+  Driver/HAL library for Semtech SX1301 multi-channel modem and
+  SX1257/SX1255 RF transceivers.
+endef
+
+define Package/libloragw/config
+	source "$(SOURCE)/Config.in"
+endef
+
+define Package/libloragw-tests
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=LoRaWAN
+  TITLE:=Test programs for libloragw to check functionality
+  DEPENDS:=@libloragw
+endef
+
+define Package/libloragw-utils
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=LoRaWAN
+  TITLE:=Utility programs for libloragw
+  DEPENDS:=@libloragw
```

Comment:
Replace ```@``` with ```+```

Reply:
Just curious, what's the actual difference between `@` and `+`?

Reply:
@brainstorm 
Please have a look at https://openwrt.org/docs/guide-developer/packages Section ´Dependency Types´